requirements/customer/scheduling-algorithm.tex

```diff
@@ -0,0 +1,48 @@
+\subsection{Scheduling Algorithm}
+This section describes how the central server distributes jobs to work machines: the \textit{scheduling algorithm}.
+Put briefly, the scheduling algorithm takes a list of jobs and decides which ones to run on the available work machines.
+The scheduling algorithm has the following goals:
+\begin{itemize}
+\item Keep work machines busy: there should be no idle work machines if there are pending jobs that could be run on them.
+\item Consider job priorities: jobs with high priorities should be worked on first.
+\item Consider time of job submission: jobs that were submitted first should be worked on first (if priorities are equal).
```

Comment:
As discussed today, maybe this and the previous item need to be changed (so, say 6 hours in queue increases priority level by 1).